package.json

```diff
@@ -27,7 +27,7 @@
     "localforage": "1.5.6",
     "localforage-observable": "1.4.0",
     "marked": "mattermost/marked#802e981ade71149a497cbe79d12b8a3f82f7657e",
-    "mattermost-redux": "mattermost/mattermost-redux",
+    "mattermost-redux": "mattermost/mattermost-redux#58adb9a71cf5b3d1b753f33295da81372702cd45",
```

Comment:
Is this intentional? I thought you wanted to switch to not including the hash here

Reply:
I'll remove it from here, good catch